specification/ServiceBusManagement.json

```diff
@@ -0,0 +1,413 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-04",
+    "title": "ServiceBusManagementClient",
+    "description": "The Azure ServiceBus Management API."
+  },
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/xml"
+  ],
+  "consumes": [
+    "application/xml"
+  ],
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{namespaceName}.servicebus.windows.net",
+    "parameters": [
+      {
+        "name": "namespaceName",
+        "description": "The name of the ServiceBus namespace to send requests to.",
+        "required": true,
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true
+      }
+    ]
+  },
+  "paths": {
+    "/{queuePath}/": {
+      "get": {
+        "tags": [
+          "Queues"
+        ],
+        "operationId": "Queue_Get",
```

Comment:
Please make sure that the entry in the tags array matches the initial part in the operationId. The operationId should also be "Queues_Get". We recommend having plural nouns for the operation group.